charts/hivemq-swarm/values.yaml

```diff
@@ -0,0 +1,240 @@
+  ##
+  volumeClaimTemplates: [ ]
+
+## HiveMQ Swarm Agents configuration
+##
+agents:
+  logLevel: info
+  agentCount: 3
+
+  ## Configure resources for HiveMQ Swarm Agent Pods (Note: request = limits for HiveMQ Swarm)
+  ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+  ##
+  resources:
+    cpu: 500m
+    memory: 1G
+
+  ## Configure Pods Security Context
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-pod
+  ##
+  podSecurityContext:
+    sysctls:
+      - name: net.ipv4.ip_local_port_range
+        value: "1024 65535"
+
+  ## Configure HiveMQ Swarm Agent Container Security Context
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-container
+  ##
+  securityContext: {}
+
+  ## Configure affinity preset for HiveMQ Swarm Agent Pods
+  ## ref: https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node
+  ##
+  affinity: {}
+    # podAntiAffinity:
+    #   preferredDuringSchedulingIgnoredDuringExecution:
+    #     # Try not to co-locate with other swarm instances
+    #     - weight: 50
+    #       podAffinityTerm:
+    #         labelSelector:
+    #           matchExpressions:
+    #             - key: swarm-task
+    #               operator: In
+    #               values:
+    #                 - ${LABEL}
+    #         topologyKey: "kubernetes.io/hostname"
+    #     # Try not to co-locate with HiveMQ nodes
+    #     - weight: 60
+    #       podAffinityTerm:
+    #         labelSelector:
+    #           matchExpressions:
+    #             - key: app
+    #               operator: In
+    #               values:
+    #                 - hivemq
+    #         topologyKey: "kubernetes.io/hostname"
+  # Add environment variables for all agents
+  env: [ ]
+
+  ## Additional volumes for the HiveMQ Swarm agent pod.
+  extraVolumes: [ ]
+
+  ## Extra volume mounts that will be added to the swarm agent container
+  extraVolumeMounts: [ ]
+
+## Monitoring configuration
+##
+monitoring:
+  # Deploy ServiceMonitor for Prometheus Operator and enable Prometheus
+  enabled: false
+  # Deploy a dedicated instance of the prometheus operator, including grafana, as sub-chart
+  dedicated: false
+  # dashboardNamespace: monitoring (defaults to use the namespace this chart is deployed to)
+  serviceMonitor:
+    interval: 15s
+    #  namespace: monitoring  (defaults to use the namespace this chart is deployed to)
+    scrapeTimeout: 10s
+  
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+## Pod annotations
+## ref: https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/
+##
+podAnnotations: {}
```

Comment:
Just noticed this - we set this here "globally" for both agent and commander - maybe we should move this into the individual sections?